README.md

```diff
@@ -126,6 +126,22 @@ All executable fields, such as `onStart` and `onChange`, accept both a string or
 ]
 ```
 
+### Template Configuration
+
+Containerbuddy configuration has template support, such as substituting environment variables in the config file. This makes it possible to use the same configuration and alter the behavior per environment. Full documentation about the template language used can be found in the [Go text/template Docs](https://golang.org/pkg/text/template/).
```

Comment:
I feel like we should be a bit more straightforward about how environment variables are interpreted. The `text/template` docs are accurate but are dense and talk about "fields" (which is an implementation detail) but what we're really trying to say is "if you set an environment variable `FOO=BAR` then you can read it into the template like `{{ .FOO }}`". We should make that common case front-and-center of this section and then follow up with something along the lines of "oh and this is Go `text/template` language if you want more".